knife-ec2.gemspec

```diff
@@ -19,10 +19,10 @@ Gem::Specification.new do |s|
   s.add_dependency 'fog',           '~> 1.25.0'
   s.add_dependency 'knife-windows', '>= 0.8.2'
 
-  s.add_development_dependency 'mixlib-config', '~> 2.0'
-  s.add_development_dependency 'chef',          '>= 11.16.2', '< 12'
-  s.add_development_dependency 'rspec',         '~> 2.14'
-  s.add_development_dependency 'rake',          '~> 10.1'
+  s.add_development_dependency 'mixlib-config'
+  s.add_development_dependency 'chef'
+  s.add_development_dependency 'rspec'
+  s.add_development_dependency 'rake'
   s.add_development_dependency 'sdoc',          '~> 0.3'
```

Comment:
Hi,

Just wanted to say hi and tell you how much I miss you, here are some interesting things for you <http://ntiprorytho.mikesdogshack.info/oitajwc>

Be well, hebgb@blueyonder.co.uk